docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md

```diff
@@ -127,13 +151,15 @@ YugabyteDB Managed clusters support 15 simultaneous connections per vCPU. So a c
 | 6x4 | 360 |
 | 6x8 | 720 |
 
+For production clusters, a minimum of 3 nodes with 4 to 8 vCPUs per node is recommended.
+
 During an update, one node is always offline. When sizing your cluster to your workload, ensure you have enough additional capacity to support rolling updates with minimal impact on application performance. You can also mitigate the effect of updates on performance by [scheduling them](../../cloud-clusters/cloud-maintenance/) during periods of lower traffic.
 
-YugabyteDB Managed supports both vertical and horizontal scaling. Depending on your performance requirements, you can increase the number of vCPUs per node, as well as the total number of nodes. You can also increase the disk size per node. However, once increased, you can't lower the disk size per node.
+If your configuration doesn't match your performance requirements, you can scale your cluster after it is created. Depending on your performance requirements, you can increase the number of vCPUs per node (scale up, also referred to as vertical scaling), as well as the total number of nodes (scale out, also referred to as horizontal scaling). You can also increase the disk size per node. However, once increased, you can't lower the disk size per node.
 
-If your configuration doesn't match your performance requirements, you can change these values after the cluster is created (increasing or decreasing vCPUs and increasing storage, and adding or removing nodes). Refer to [Scaling clusters](../../cloud-clusters/configure-clusters/).
+To realize the full benefit from scaling a cluster, for typical workloads you should scale up to 8 or 16 vCPUs before adding additional nodes. For example, for a 3-node cluster with 4 vCPUs per node, scale up to 8 or 16 vCPUs before adding a fourth node.
```

Comment:
I would simplify and make more direct: For example:

```
To realize the full benefit from scaling a cluster, for typical workloads you should scale up to 8 or 16 vCPUs before adding additional nodes. For example, for a 3-node cluster with 4 vCPUs per node, scale up to 8 or 16 vCPUs before adding a fourth node.
```

becomes

```
YugabyteDB recommends vertical scaling until nodes have 16vCPUs, and horizontal scaling once nodes have 16 vCPUs. 

For example, for a 3-node cluster with 4 vCPUs per node, scale up to 8 vCPUs rather than adding a 4th node. 
For a 3-node cluster with 16 vCPUs per node, scale out by adding a 4th node.
```